src/wizard/interfaces.ts

```diff
@@ -48,13 +48,18 @@ export interface WizardProps extends BaseComponentProps {
    * - `cancelButton` (string) - The text of the button that enables the user to exit the flow.
    * - `previousButton` (string) - The text of the button that enables the user to return to the previous step.
    * - `nextButton` (string) - The text of the button that enables the user to move to the next step.
-   * - `submitButton` (string) - The text of the button that enables the user to submit the form.
+   * - `submitButton` (deprecated) (string) - The text of the button that enables the user to submit the form.
```

Comment:
There are two other cases of deprecated nested props [here](https://github.com/cloudscape-design/components/blob/main/src/flashbar/interfaces.ts#L60) and [here](https://github.com/cloudscape-design/components/blob/main/src/flashbar/interfaces.ts#L62), and they use a different formatting. And I think that other formatting brings the opportunity to state what other property should be used instead (as you already did in the prop further down, but that string is not shown in our API docs).